test/integration/http-service.yaml.tmpl

```diff
@@ -43,8 +43,6 @@ spec:
           - "{{.port1}}"
           - --port
           - "{{.port2}}"
-          - --version
```

Comment:
Intentional? I didn't see anything in the commit summary about removing versioned service.